lib/index.js

```diff
@@ -58,10 +63,10 @@ function getRealmConstructor(context) {
     }
 }
 
-const context = utils.getContext();
-const realmConstructor = getRealmConstructor(context);
+const environment = utils.getEnvironment();
+const realmConstructor = getRealmConstructor(environment);
 
-require("./extensions")(realmConstructor, context);
+require("./extensions")(realmConstructor, environment);
```

Comment:
Perhaps [the local variable on the other side of this call](https://github.com/realm/realm-js/blob/ff/metrics_s3/lib/extensions.js#L68) should be renamed from `context` to `environment` as well, to keep naming consistent?